pom.xml

```diff
@@ -99,6 +100,25 @@ THE SOFTWARE.
         <artifactId>args4j</artifactId>
         <version>2.33</version>
       </dependency>
+      <dependency>
+        <groupId>commons-io</groupId>
+        <artifactId>commons-io</artifactId>
+        <version>2.6</version>
+      </dependency>
+
+      <!--TODO(oleg-nenashev): Remove once in BOM -->
+      <dependency>
+        <groupId>org.jenkins-ci.plugins.workflow</groupId>
+        <artifactId>workflow-multibranch</artifactId>
+        <version>2.21</version>
+      </dependency>
+      <!--TODO(oleg-nenashev): Is it a valid use-case? Move to BOM if so -->
+      <dependency>
+        <groupId>org.jenkins-ci.plugins.workflow</groupId>
+        <artifactId>workflow-scm-step</artifactId>
+        <classifier>tests</classifier>
```

Comment:
Why is this even here? I would delete it.

Reply:
I might. Need to check WTF it is first. I still discover various hacks related to the former usage of JTH inside JFR

Reply:
Which ones? I thought I had decoupled completely from JTH

Reply:
Yes an no. There is no dependencies on JTH specifically, but there are bits like https://issues.jenkins-ci.org/browse/JENKINS-55764 which likely require a follow-up.